src/repl/reader.rs

```diff
@@ -46,8 +46,10 @@ pub(super) fn parse_num<T: Default + TryFrom<u128>>(num: &str) -> Result<T> {
     let (radix, numstr) = match num {
         "0" => return Ok(T::default()),
         s if s.starts_with("0x") || s.starts_with("0X") => (16, &s[2..]),
+        s if s.ends_with("h") || s.ends_with("H") => (16, &s[..s.len() - 1]),
         s if s.starts_with("0t") || s.starts_with("0T") => (10, &s[2..]),
         s if s.starts_with("0b") || s.starts_with("0B") => (2, &s[2..]),
```

Comment:
Could we also have `0i` for binary (like mdb)?

Reply:
Sure! I didn't even know that was thing. Give me a moment.

Reply:
PTAL.